packages/bsky/src/hydration/graph.ts

```diff
@@ -162,7 +158,11 @@ export class GraphHydrator {
     const blocks = new Blocks()
     for (let i = 0; i < deduped.length; i++) {
       const pair = deduped[i]
-      blocks.set(pair.a, pair.b, res.exists[i] ?? false)
+      const block = res.blocks[i]
+      blocks.set(pair.a, pair.b, {
+        blockUri: block.blockedBy || block.blocking || undefined,
+        blockListUri: block.blockedByList || block.blockingByList || undefined,
+      })
```

Comment:
May consider defensively assuming `block` might not exist, even though it should.  Could log a warn or error in that case.